specification/search/data-plane/Azure.Search/preview/2024-03-01-Preview/searchindex.json

```diff
@@ -1868,6 +1868,11 @@
         "exhaustive": {
           "type": "boolean",
           "description": "When true, triggers an exhaustive k-nearest neighbor search across all vectors within the vector index. Useful for scenarios where exact matches are critical, such as determining ground truth values."
+        },
+        "oversampling": {
+          "type": "number",
+          "format": "double",
+          "description": "Oversampling factor. Minimum value is 1. It overrides the defaultOversampling parameter configured in the index definition. It can be set only when 'rerankWithOriginalVectors' is true. This parameter is only permitted when a compression method is used on the underlying vector field."
```

Comment:
```suggestion
          "description": "Oversampling factor. Minimum value is 1. It overrides the 'defaultOversampling' parameter configured in the index definition. It can be set only when 'rerankWithOriginalVectors' is true. This parameter is only permitted when a compression method is used on the underlying vector field."
```